### ISSUE TITLE
Uncaught (in promise) TypeError: wellKnownResponse is undefined

### ISSUE THREAD

**Issue by xkabr00** [okta/okta-auth-js#1499]
### Describe the bug

Trying to setup Okta connection for Nuxt 3 SPA application, having double checked the settings in Okta admin - allowed urls, redirects etc.

I am currently encountering an error "Uncaught (in promise) TypeError: wellKnownResponse is undefined" when calling signInWithRedirect().

The GET request to /.well-known/openid-configuration returns 200 with code_challenge_methods_supported correctly set.

I have implemented custom httpRequestClient to avoid another issue with CORs and x-okta-user-agent-extended header.

![2024-03-08 213209](https://github.com/okta/okta-auth-js/assets/28796220/46b0fe4d-93df-4a85-a9c9-999ca7949a4a)

### Reproduction Steps?

npx nuxi@latest init
add **@okta/okta-auth-js 7.5.1** and **@okta/okta-vue 5.7.0**

Okta auth config:
```
import { OktaAuth } from '@okta/okta-auth-js'

const customHttpRequestClient = async (method, url, args) => {
  if (args.headers) {
    delete args.headers['x-okta-user-agent-extended'];
    delete args.headers['X-Okta-User-Agent-Extended'];
  }

  const response = await fetch(url, {
    method: method,
    headers: args.headers,
    body: JSON.stringify(args.data),
    credentials: 'omit'
  });

  if (response.ok) {
    return response.json();
  }

  throw new Error(`HTTP error! status: ${response.status}`);
};

const oktaAuth = new OktaAuth({
  issuer: `https://${import.meta.env.VITE_OKTA_URL}`,
  clientId: `${import.meta.env.VITE_OKTA_CLIENT_ID}`,
  redirectUri: window.location.origin + '/login-callback',
  scopes: ['openid', 'profile', 'email'],
  httpRequestClient: customHttpRequestClient,
  pkce: true
})

export { oktaAuth }
```

### SDK Versions

System:
    OS: Windows 10 10.0.19045
    CPU: (12) x64 Intel(R) Core(TM) i5-10400F CPU @ 2.90GHz
    Memory: 7.18 GB / 15.89 GB
  Binaries:
    Node: 18.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Chromium (122.0.2365.66)
    Internet Explorer: 11.0.19041.3636

**Comment by jaredperreault-okta** [comment]
@xkabr00 Do you no longer need this?

**Comment by xkabr00** [comment]
I have solved it by ditching @okta/okta-auth-js and @okta/okta-vue altogether and used @auth0/auth0-vue instead

**Comment by jaredperreault-okta** [comment]
I'm glad your unblocked. Do you mind sharing what lead you to this decision? We are always looking for feedback
